java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -18,45 +18,22 @@
 
 package ai.rapids.cudf;
 
-import ai.rapids.cudf.HostColumnVector.BasicType;
-import ai.rapids.cudf.HostColumnVector.Builder;
-import ai.rapids.cudf.HostColumnVector.DataType;
-import ai.rapids.cudf.HostColumnVector.ListType;
-import ai.rapids.cudf.HostColumnVector.StructData;
-import ai.rapids.cudf.HostColumnVector.StructType;
-
+import ai.rapids.cudf.HostColumnVector.*;
```

Comment:
In most coding styles star imports are discouraged not worth changing in an unrelated PR. If it's your IDE, you can change it in settings.

Reply:
Thanks Gera. I just reverse back. Yeah my IDE automatically did that. Personally, I prefer `import *` as it makes thing looks much cleaner.